dsindex/indexer.go

```diff
@@ -267,11 +267,11 @@ func encode(data string) string {
 	if data == "" {
 		return ""
 	}
-	return base36.EncodeToStringUc([]byte(data))
+	return base64.URLEncoding.WithPadding(base64.NoPadding).EncodeToString([]byte(data))
 }
 
 func decode(data string) (string, error) {
-	b, err := base36.DecodeString(data)
+	b, err := base64.URLEncoding.WithPadding(base64.NoPadding).DecodeString(data)
```

Comment:
As a way of future proofing, do we want to pay an extra byte and just use go-multibase...?
cc @aschmahmann

Reply:
It is only one more byte, so I am going to vote yes and update PR.

Reply:
I think is good in case we want to change the encoding at some point, but we still need to be careful not to choose a format that will bream how keys are stored.  For example, if we chose standard Base64 encoding, this would cause problems because "/" is used as a delimiter in stored keys.

Reply:
@aschmahmann Want to cast the final vote?

Reply:
TLDR: Sure, why not use multibase.

Pros:
- if gives us a bit of flexibility in the future
- could make tooling that pokes around at the datastore a little easier to work with
- if we were using multibase everywhere in the datastore (except maybe the blockstore) things would be a lot easier to inspect

Cons:
- might encourage someone to think it's ok to use different encodings
- might confuse people when they see CIDv0's encoded as multibase but missing the version + codec
- costs us 1 byte per key

Neither the pros nor the cons are particularly expensive, however it'd be nice to start building some datastore introspection tooling and this could be helpful.